Content.Client/Construction/UI/ConstructionMenuPresenter.cs

```diff
@@ -223,15 +223,15 @@ private void OnViewPopulateRecipes(object? sender, (string search, string catago
             _constructionView.RecipesGridScrollContainer.Visible = _constructionView.GridViewButtonPressed;
             _constructionView.Recipes.Visible = !_constructionView.GridViewButtonPressed;
 
+            var spriteSystem = _systemManager.GetEntitySystem<SpriteSystem>();
```

Comment:
SpriteSystem should be listed as a dependency at the top of the file with the rest of them

Reply:
Also noted that on line 380, it's still calling the obsolete Frame0 property on the recipe icon